_redirects

```diff
@@ -159,6 +160,9 @@
 /docs/building-a-dbt-project/using-sources /docs/build/sources 301
 /docs/building-a-dbt-project/projects /docs/build/projects 301
 /docs/building-a-dbt-project/building-models/python-models /docs/build/python-models 301
+/docs/building-a-dbt-project/macros  /docs/guides/building-packages 301
+/docs/building-a-dbt-project/setting-up /docs/guides/building-packages 301
+docs/building-a-dbt-project/dbt-jinja-functions /docs/guides/building-packages 301
```

Comment:
There is a missing `/` at the start of this redirect, which may cause this one to not work as expected.